BuildResidentialHPXML/measure.rb

```diff
@@ -3432,7 +3435,12 @@ def arguments(model) # rubocop:disable Lint/UnusedMethodArgument
     return args
   end
 
-  # define what happens when the measure is run
+  # Define what happens when the measure is run.
+  #
+  # @param model [OpenStudio::Model::Model] OpenStudio Model object
+  # @param runner [OpenStudio::Measure::OSRunner] OpenStudio Runner object
+  # @param user_arguments [OpenStudio::Measure.convertOSArgumentVectorToMap] OpenStudio::Measure.convertOSArgumentVectorToMap object
```

Comment:
According to `user_arguments.class`, this object is a `OpenStudio::Measure::OSArgumentMap` object.

```suggestion
  # @param user_arguments [OpenStudio::Measure::OSArgumentMap] OpenStudio measure arguments
```